packages/uniorg-parse/src/parser.ts

```diff
@@ -762,17 +768,21 @@ class Parser {
 
       valueLines.push(this.r.advance(this.r.line()));
     }
-
     let value = valueLines.join('');
     if (value[value.length - 1] === '\n') {
       value = value.substring(0, value.length - 1);
     }
+    const end = begin + value.length;
```

Comment:
❗🐛 this calculation is incorrect for multi-line comments because it does not take into account leading space, `#` and trailing newline.

If you can, it's always better to use `this.r.offset()` instead of trying to calculate the end position manually.
```suggestion
    const end = this.r.offset();
```